api/turing/server/application.go

```diff
@@ -153,3 +148,49 @@ func Run() {
 		log.Errorf("Failed to start turing-api: %s", err)
 	}
 }
+
+func initDB(cfg *config.Config) (*gorm.DB, error) {
```

Comment:
It makes sense to move it into a separate package (`db`?) and re-use this part for the integration tests db initialization (https://github.com/gojek/turing/blob/main/api/turing/it/database/database.go)

Reply:
Also, you don't need the entire `*config.Config` here, you can simply pass an instance of `config.DatabaseConfig` instead

Reply:
Ok db package sounds better actually.